adrs/2022-10-17-runner-image.md

```diff
@@ -3,6 +3,19 @@
 
 **Status**: Done
 
+# Breaking Changes
```

Comment:
This ADR is really outdated now... 
Should we make another one and mark this one as outdated?